cmd/validator/validator.go

```diff
@@ -23,6 +23,10 @@ optional flags:
     	Format of the printed report. Options are standard and json (default "standard")
   -version
     	Version prints the release version of validator
+  -additional-file
```

Comment:
Since we're expecting a comma-separated list it probably makes sense to name it `-additional-files`